model/rpc/common.yaml

```diff
@@ -0,0 +1,33 @@
+groups:
+  - id: attributes.metrics.rpc.server
+    type: attribute_group
+    brief: "Describes RPC metric attributes."
+    attributes:
+      - ref: rpc.system
+        requirement_level: required
+      - ref: rpc.service
+        note: ""
+      - ref: rpc.method
+        note: ""
+      - ref: network.transport
+      - ref: server.address
+        note: ""
+        requirement_level: opt_in
+      - ref: server.port
+        note: ""
+        requirement_level: opt_in
+      - ref: error.type
+        requirement_level:
+          conditionally_required: If and only if the operation failed.
+  - id: attributes.metrics.rpc.client
+    type: attribute_group
+    brief: "Describes RPC metric attributes."
+    extends: attributes.metrics.rpc.server
+    attributes:
+      - ref: server.address
+        note: When communicating through an intermediary, `server.address` SHOULD represent the server address behind any intermediaries, for example proxies, if it's available.
```

Comment:
this is essentially a copy-paste of note in the definition, do we need to rewrite it? I see the reason we want to have custom note for server-side, can we write it in the way that we don't erase the original note and then have to bring it back?